src/pages/CustomRichTextEditorLink.tsx

```diff
@@ -143,7 +151,7 @@ function CustomRichTextEditorLink({
             Component={
               <Input
                 onChange={updateValue}
-                onBlur={updateValue}
+                onBlur={(event) => {}}
```

Comment:
I see you have a no-op here now, is this intentional?

Reply:
Oops not just leftover tinkering to check the proper types